### PR TITLE
Fix LableBMFont is not use padding

### PR DESCRIPTION
**Pull request by waiter**
LableBMFont have not use padding, so the word space is not correct. 
if the padding is very big, you will see the error.

I fix it in the cocos2d-x 2.2, I have not test 3.0, but I think it should be like that!

### CODE REVIEW

**Comment by minggo** [comment]
Thanks for the patch.
We will use new Label to re-write LabelBMFont.

So how about close this pull request and polish it with new Label?
Thanks.

**Comment by waiter** [comment]
@minggo 
What does the "New Label" mean?  I pull the request to the Develop branch. And I modify the "CCFontFnt.cpp" in this branch. 
And another pull request is pulled to the Master branch ( https://github.com/cocos2d/cocos2d-x/pull/5591 )

File: cocos/2d/CCFontFNT.cpp [hunk 1]
```diff
@@ -413,7 +413,7 @@ void BMFontConfiguration::purgeFontDefDictionary()
              */
 
             uint16_t lineHeight = 0; memcpy(&lineHeight, pData, 2);
-            _commonHeight = lineHeight;
+            _commonHeight = lineHeight - _padding.top - _padding.bottom;
```
Comment (WenhaiLin) [inline cocos/2d/CCFontFNT.cpp:416]: Is the padding  increase line height  in version 1.9【Block type 2】?The padding only increase size of character in version 1.13. I can't found old version for test it.

Comment (waiter) [inline cocos/2d/CCFontFNT.cpp:416]: @Dhilan007 
I have no idea with "Block type 2". 
I use "Hiero V3.0" to make FNT file. And I test in cocos2d-x 2.2, The padding increase the line height.

**Comment by cocosQA** [comment]
Dear waiter:
  We have created a new branch 'v3' to replace branch 'develop', and this PR has been moved from 'develop' branch to 'v3' branch, the new PR is https://github.com/cocos2d/cocos2d-x/pull/6454
The old 'develop' branch will be deleted soon, and this PR will be closed accordingly.

**Pull request closed without merging**